src/Authentication/Login.php

```diff
@@ -118,7 +118,11 @@ protected function getResults(\stdClass $response, $type) : array
         if (!empty($this->auth['apiRequest'][$type])) {
             foreach ($this->auth['apiRequest'][$type] as $key => $path) {
                 try {
-                    $result[$key] = \Keboola\Utils\getDataFromPath($path, $response, '.', false);
+                    if ($type === 'headers' && is_array($path)) {
```

Comment:
no z kodu vyzeralo, ze tam moze byt len `headers` alebo `query` ... ale nevedel som dohladat nejaky priklad na to query, tak som to nechal len pre headers ...